### ISSUE TITLE
Coverage result of zero included for set of variables "vars"

### ISSUE THREAD

**Issue by lemmy** [tlaplus/tlaplus#152]
```tla
--------- MODULE Foo ----------
VARIABLES x, y
vars == <<x, y>>

...

=======================
```

"vars" is obviously no action and thus reporting coverage is slightly confusing.

**Comment by lemmy** [comment]
Related to issue #92

**Comment by lemmy** [comment]
Fixed by b6c763f813582cab49f5b94a5a48ad34971f5c23
